stores/benchmark.ts

```diff
@@ -118,15 +118,13 @@ export function getLastBenchmarkColor(): string {
   return formatHex(oklch)
 }
 
-export function trackPaint(
+export function reportPaint(
   type: RenderType,
+  ms: number,
   isFull: boolean,
-  cb: () => void
+  isWorker?: boolean
```

Comment:
It is dirty. Just call `reportFreeze` in the place when you call `reportPaint` without worker.